bundle/jsky.app.ot/src/main/java/jsky/app/ot/editor/seq/DynamicSequenceTableModel.java

```diff
@@ -166,7 +162,7 @@ public void setSequence(ConfigSequence sequence, SPNodeKey key, List<ItemKey> al
             res.addAll(groupedItems.get(category));
         }
 
-        return res.toArray(new ItemKey[0]);
```

Comment:
The warning IntelliJ gave me said that in this case reflection was used to create an array of the appropriate type and size which should be avoided for efficiency reasons.